src/main/resources/application-prod.yml

```
@@ -153,13 +147,9 @@ ai:
-app:
-  base-url: ${BASE_URL} # 운영 환경 도메인
-
-
+# 🌍 CORS 설정
 cors:
   allowed-origins:
-    - "https://ticket-mon.o-r.kr"          # 프론트 도메인
+    - "https://ticket-mon.o-r.kr"          # 프론트엔드 도메인
     - "https://api.ticket-mon.o-r.kr"      # API 도메인
-    - "http://localhost:3000"              # 로컬 개발용 프론트 도메인
+    - "http://localhost:3000"              # 로컬 개발용 프론트엔드 도메인
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)